### ISSUE TITLE
tried multiple modes but fail to get better resolution

### ISSUE THREAD

**Issue by liewsanmin** [alexjc/neural-enhance#109]
Hi,

I am new to deep learning. I tried enhancing the same sample shown on the GitHub page with the following modes: 
deblur 1x, zoom 2x, and repair 1x. 
version of trained model: 0.3

But the resulting image is still not as enhance as the one showed in the sample. Is there anything I am missing?

Original:
![image](https://cloud.githubusercontent.com/assets/12568152/20852544/e8d65350-b8ab-11e6-832b-1c0eec17a5d6.png)

Repair 1x:
![image](https://cloud.githubusercontent.com/assets/12568152/20852493/bfcade40-b8ab-11e6-8293-f521da279ac5.png)

zoom 2x:
![image](https://cloud.githubusercontent.com/assets/12568152/20852513/ce29bbf0-b8ab-11e6-8722-f819419d3401.png)

deblur 1x:
![image](https://cloud.githubusercontent.com/assets/12568152/20852535/e1da2716-b8ab-11e6-903b-0f18ef875fed.png)

**Comment by alexjc** [comment]
The input image you're giving it is artificially resized to have 8x size pixels. (I presume you just used the example, it's shown this way to make the image visible in the README.) You need to feed in the original 1x image, not artificially resized.

**Comment by liewsanmin** [comment]
Hi Alex,

Thanks!! I explored around your README and tried the main website in [http://enhance.nucl.ai/](URL). And then I realized I was inputting resolutions as input images. I scaled down images "equivalent to 256x256 or 320x200" as the website states, and I see immediate effect with just 2x (4x will take forever with my MacBookPro late 2015)

Here is the example of my circuit board image zoomed: 
![image](https://cloud.githubusercontent.com/assets/12568152/20861930/e442eb2a-b962-11e6-9d5e-64454ea91fe9.png)

Image enhance 2x : 
![image](https://cloud.githubusercontent.com/assets/12568152/20861935/f0c037f4-b962-11e6-8236-4570461be36a.png)

I can notice the edges are much smoother than the 1st one. 

Keep up the good work!
